server/src/main/java/org/opensearch/cluster/ClusterInfo.java

```diff
@@ -68,9 +69,12 @@ public class ClusterInfo implements ToXContentFragment, Writeable {
     final Map<ShardRouting, String> routingToDataPath;
     final Map<NodeAndPath, ReservedSpace> reservedSpace;
     final Map<String, FileCacheStats> nodeFileCacheStats;
+    private long avgTotalBytes;
+    private long avgFreeByte;
 
     protected ClusterInfo() {
         this(Map.of(), Map.of(), Map.of(), Map.of(), Map.of(), Map.of());
+        calculateAvgFreeAndTotalBytes(mostAvailableSpaceUsage);
```

Comment:
We don't need to compute again here since the `this` invocation will call the other ctor?

Reply:
Ack. Will remove this.

Reply:
Addressed your comments